components/Footer.js

```diff
@@ -172,7 +174,7 @@ function Footer({
             </Link>
             &
             <Link route={RouteNames.Datenschutz} passHref>
-              <Anchor> {datenschutzText}</Anchor>
+              <Anchor>{datenschutzText}</Anchor>
```

Comment:
Removing this space will remove the space between the `&` and `Datenschutz` text.

Reply:
Ooops! Good catch.